.eslintrc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "eslint:recommended",
+  "extends": ["eslint:recommended", "plugin:react/recommended"],
```

Comment:
should we add `"plugin:react/recommended"` only for examples folder?

Reply:
Agree. There are no react code in packages folder.